stylesheets/_component.datatables.scss

```diff
@@ -373,3 +373,23 @@ th.sorting_disabled:hover {
 }
 
 // scss-lint:enable SelectorFormat
+
+.dtr-details {
+    .dtr-title {
+        a {
```

Comment:
Is this block to address the same issue as below (underlined links?) if so why are the extra `color` and `font-family` styles needed?

Reply:
colour and font family are used to switch styles when DT goes into the (+) responsive mode, because the links move out of the `th` they adopt regular `td` styles when we want them to still look like headers, they also lose their interactive features so if I remember rightly, Nikos and I discussed just resetting the styles like this.

Reply:
👍